packages/cli/src/lib/log.ts

```diff
@@ -166,6 +169,33 @@ export function muteDevLogs({workerReload}: {workerReload?: boolean} = {}) {
       () => {},
     ],
   );
+
+  addMessageReplacers(
+    'dev-vite',
+    // Vite logs
+    [
+      // This log must come from Rollup and does not go through Vite's customLogger
+      ([first]) =>
+        typeof first === 'string' && /^Sourcemap for .*@remix-run/i.test(first),
+      () => {},
+    ],
+    [
+      // Log generated from Workerd HMR connection to Vite
+      ([first]) =>
+        typeof first === 'string' &&
+        /^\[vite\] (connected|program reload)/i.test(first),
+      () => {},
+    ],
+    [
+      // Log that gets entangled with our initial dev logs
+      ([first]) =>
+        typeof first === 'string' &&
```

Comment:
I know we want to keep the logs clean, but does it ever concern you just how much we monkey console.log and family?

Reply:
Ah, I see this other PR: https://github.com/Shopify/hydrogen/pull/1928

This makes me much more comfortable.